mmv1/third_party/terraform/services/resourcemanager/resource_google_project.go

```diff
@@ -68,6 +68,7 @@ func ResourceGoogleProject() *schema.Resource {
 			},
 			"skip_delete": {
 				Type:        schema.TypeBool,
+				Deprecated:  `'skip_delete' is deprecated and will be removed in a future major release. The new release adds support for 'deletion_policy' instead.`,
```

Comment:
I think you can remove quotation marks in this message and it should fix the compilation error

Reply:
Thanks, removed quotation marks. Doesn't seem to help though.

Reply:
Locally the provider generation works fine. There's something wrong with the test, it's picking up older version perhaps.